src/scripts/main.js

```diff
@@ -1,7 +1,129 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+function keyListener(e) {
+  if (game.getStatus() !== 'playing') {
+    return;
+  }
 
-// Write your code here
+  switch (e.key) {
+    case 'ArrowDown':
+      game.moveDown();
+      break;
+    case 'ArrowLeft':
+      game.moveLeft();
+      break;
+    case 'ArrowRight':
+      game.moveRight();
+      break;
+    case 'ArrowUp':
+      game.moveUp();
+      break;
+  }
+
+  update();
+}
```

Comment:
Consider debouncing the keyListener function to improve performance and prevent potential issues with rapid key presses. This is especially useful if the game logic becomes more complex or if the `update` function is computationally expensive.